src/main/java/io/stargate/sgv2/jsonapi/service/operation/filters/table/codecs/JSONCodec.java

```diff
@@ -128,6 +128,10 @@ public boolean testToJSON(DataType fromCQLType) {
    *     should convert this into the appropriate exception for the use case.
    */
   public JsonNode toJSON(ObjectMapper objectMapper, CqlT value) throws ToJSONCodecException {
+    // First: if caller passes nulls, convert to JSON Null so converter need to do null checks
```

Comment:
This just to make sure all codecs accept `null`; caller may or may not handle it instead of handing to codecs (in case it wants to suppress them, for example).